genrl/deep/common/VecEnv.py

```diff
@@ -178,22 +201,26 @@ def render(self, mode="human"):
 images in 'human' and returns tiled images in 'rgb_array'
         :type mode: string
         """
-        images = np.asarray(self.images())
-        N, H, W, C = images.shape
-        newW, newH = int(np.ceil(np.sqrt(W))), int(np.ceil(np.sqrt(H)))
-        images = np.array(list(images) + [images[0] * 0 for _ in range(N, newH * newW)])
-        out_image = images.reshape(newH, newW, H, W, C)
-        out_image = out_image.transpose(0, 2, 1, 3, 4)
-        out_image = out_image.reshape(newH * H, newW * W, C)
-        if mode == "human":
-            import cv2  # noqa
-
-            cv2.imshow("vecenv", out_image[:, :, ::-1])
-            cv2.waitKey(1)
-        elif mode == "rgb_array":
-            return out_image
-        else:
-            raise NotImplementedError
+        self.envs[0].render()
+
+        # Does not work need to debug how can vectorized envs be rendered
+        # images = np.asarray(self.images())
+        # N, H, W, C = images.shape
+        # newW, newH = int(np.ceil(np.sqrt(W))), int(np.ceil(np.sqrt(H)))
+        # images = np.array(list(images) + [images[0] * 0 for _ in range(N, newH * newW)])
+        # out_image = images.reshape(newH, newW, H, W, C)
+        # out_image = out_image.transpose(0, 2, 1, 3, 4)
+        # out_image = out_image.reshape(newH * H, newW * W, C)
+        # if mode == "human":
+        #     # make_grid(self.images())
+        #     import cv2  # noqa
+
+        #     cv2.imshow("vecenv", out_image[:, :, ::-1])
+        #     cv2.waitKey(1)
+        # elif mode == "rgb_array":
+        #     return out_image
+        # else:
+        #     raise NotImplementedError
```

Comment:
Why are you rendering only `envs[0]`? Were you facing some issues with the grid? If so, should an issue be opened?

Reply:
Yeah, I'm not able to render the grid. Not sure what's happening here. Raised an issue. #153